e2e/playwright/viewer/src/tests/viewer.spec.ts

```diff
@@ -0,0 +1,135 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { getUserState, test, TEST_FILES } from '@alfresco/playwright-shared';
+
+test.use({ storageState: getUserState('admin') });
+test.describe('viewer file', () => {
+  const randomFolderName = `playwright-folder-${(Math.random() + 1).toString(36).substring(6)}`;
+  const randomDocxName = TEST_FILES.DOCX.name + (Math.random() + 1).toString(36).substring(6);
+  let folderId: string;
+  let fileDocxId: string;
+
+  test.beforeAll(async ({ superAdminApiClient, fileAction, shareAction, favoritesPageAction: favoritesPageAction }) => {
+    const node = await superAdminApiClient.nodes.createNode('-my-', { name: randomFolderName, nodeType: 'cm:folder', relativePath: '/' });
```

Comment:
Based on https://github.com/Alfresco/alfresco-content-app/pull/3323 lets we call api-client-factory without superAdminApiClient function